pygsti/evotypes/densitymx/opreps.pyx

```diff
@@ -727,24 +727,6 @@ cdef class OpRepExpErrorgen(OpRep):
         return _copy.deepcopy(self)  # I think this should work using reduce/setstate framework TODO - test and maybe put in base class?
 
 
-#TODO: can add this after creating OpCRep_IdentityPlusErrorgen if it seems useful
-#cdef class OpRepIdentityPlusErrorgen(OpRep):
```

Comment:
+1 to look into this

Reply:
@sserita @coreyostrove did you do the looking? If not, maybe we can open a GitHub issue to keep track of this TODO?

Reply:
Ping @coreyostrove and @sserita. Do we want to open an issue to track this TODO, or have you resolved it to your satisfaction?

Reply:
Just looked, this should be resuscitated. If I understand the code correctly then this will result in an error when using this modelmember if the representation type is '1+L'. There is an implementation of this representation in `densitymx_slow`, so an error would get hit when using the cython module (though that is the default when present).

Reply:
Gotcha. I'll revert this change and open a new issue.